src/Libraries/RevitNodes/Elements/Room.cs

```diff
@@ -0,0 +1,220 @@
+﻿using System;
+using Autodesk.Revit.DB.Architecture;
+using DynamoServices;
+using Autodesk.DesignScript.Runtime;
+using Revit.GeometryConversion;
+using RevitServices.Persistence;
+using RevitServices.Transactions;
+using System.Collections.Generic;
+
+
+namespace Revit.Elements
+{
+    /// <summary>
+    /// Room Element
+    /// </summary>
+    [DynamoServices.RegisterForTrace]
+    public class Room : Element
+    {
+        #region Internal Properties
+
+        /// <summary>
+        /// Internal reference to the Revit Element
+        /// </summary>
+        [SupressImportIntoVM]
+        internal Autodesk.Revit.DB.Architecture.Room InternalRevitElement
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// Reference to the Element
+        /// </summary>
+        public override Autodesk.Revit.DB.Element InternalElement
```

Comment:
as @sharadkjaiswal pointed out, internal access properties do not need to be suppressed from import, but I believe we should suppress the import of the public property right below.